### PR TITLE
Update test namespaces in some System.Runtime.* assmblies

### PR DESCRIPTION
**Pull request by Clockwork-Muse**
Update test namespaces in some System.Runtime.* assemblies, per #2898 .

Usings were also cleaned up.  No other work was performed.

### CODE REVIEW

File: src/System.Runtime.Extensions/tests/Performance/Perf.Environment.cs [hunk 1]
```diff
@@ -4,7 +4,7 @@
 using Microsoft.Xunit.Performance;
 using System.Runtime.InteropServices;
 
-namespace System.Runtime.Extensions.Tests
+namespace System.IO.Path.Tests
```
Comment (stephentoub) [inline src/System.Runtime.Extensions/tests/Performance/Perf.Environment.cs]: Why is "Path" in the namespace?

Comment (Clockwork-Muse) [inline src/System.Runtime.Extensions/tests/Performance/Perf.Environment.cs]: :headdesk

Comment (stephentoub) [inline src/System.Runtime.Extensions/tests/Performance/Perf.Environment.cs]: :smile:

Comment (Clockwork-Muse) [inline src/System.Runtime.Extensions/tests/Performance/Perf.Environment.cs]: Fixed, along with Perf.Path.  I think I must have copied it from there.

**Comment by stephentoub** [comment]
@dotnet-bot test this please (#3497)

**Comment by stephentoub** [comment]
LGTM.  Thanks for the continued cleanup.

**Pull request merged**